docs/docs-next/docs/concepts/automation.md

```diff
@@ -1 +1,66 @@
-# Automation
\ No newline at end of file
+---
+title: About Automation
+---
+
+Dagster has support for several types of automation. All automation in Dagster responds to some external event.
+
+The first system, and the most basic, is the [Schedule](/guides/automation/schedules), which responds to time.
+
+[Sensors](/guides/automation/sensors) are like schedules, but they respond to an external event defined by the user.
+
+[Asset Sensors](/guides/automation/asset-sensors) are a special case of sensor that responds to changes in asset materialization
+as reported by the Event Log.
+
+Finally, the Declarative Automation system is a
+more complex system that uses conditions on the assets to determine when to execute.
+
+## About Schedules
+
+In Dagster, a schedule is defined by the `ScheduleDefinition` class, or through the `@schedule` decorator. The `@schedule`
+decorator is more flexible than the `ScheduleDefinition` class, allowing you to configure job behaviour or emit log messages
+as the schedule is processed.
+
+Schedules were one of the first types of automation in Dagster, created before the introduction of Software-Defined Assets.
+As such, you may find that many of the examples can seem foreign if you are used to only working within the asset framework.
+
+For more on how assets and ops inter-relate, read about [Assets and Ops](/concepts/assets#assets-and-ops)
+
+The `dagster-daemon` process is responsible for submitting runs by checking each schedule at a regular interval to determine
+if it's time to execute the underlying job.
+
+A schedule can be thought of as a wrapper around two pieces:
+
+- A `JobDefinition`, which is a set of assets to materialize or ops to execute.
+- A `cron` string, which describes the schedule.
+
+### Defining a schedule using `ScheduleDefinition`
```

Comment:
I don't think I did a great job with this page, but my thinking was:

- how-to guides should be code-complete examples driven by a task, that you can copy-paste and run, which cover 70% of what people do
- Concept Explanations explain a topic, and code can be used to explain that topic, but there is no expectation that a code block is self-contained or runnable.

For example, in this section, I wanted to explain the difference between using @schedule and ScheduleDefinition, and using code is part of that explanation.  

I did not fully write out the explanation for each, but that would be the idea., otherwise, I think we risk polluting our guides. 

Share your concern about it becoming a kitchen sink though. I think maybe we have grades of review/input on content possibly? Maybe how-to guides anyone can start and write, but concept pages need more involvement from docs?

Reply:
> there is no expectation that a code block is self-contained or runnable.

We might get flak for this, but I get what you mean.

And yes, docs should absolutely be involved in concept pages, including whether a topic needs one or not.